src/modules/auth/home/HomeScreen.scss

```diff
@@ -18,6 +18,18 @@
 	@media (max-width: 576px) {
 		width: 90vw;
 	}
+	.status-circle {
+		width: 0.8rem;
+		height: 0.8rem;
+		background: $secondary;
+		border-radius: 50%
+	}
+	.status-green {
+		background: $light-green;
+	}
+	.status-gray {
```

Comment:
@Aringoaway I believe you dont need this `status-gray`, since default is already gray and it is `$secondary` in `.status-circle`